tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -1185,19 +1214,6 @@ def _for_each_trt_node(self, graph_def, fn):
         if node.op == _TRT_ENGINE_OP_NAME:
           fn(node)
 
-  def _rebuild_func(self, func):
-    """Rebuild function from graph_def."""
-    rebuilt_func = wrap_function.function_from_graph_def(
-        self._converted_graph_def, [tensor.name for tensor in func.inputs],
-        [tensor.name for tensor in func.outputs])
-    rebuilt_func.graph.structured_outputs = nest.pack_sequence_as(
-        func.graph.structured_outputs, rebuilt_func.graph.structured_outputs)
-    # Copy structured input signature from original function (used during
-    # serialization)
-    rebuilt_func.graph.structured_input_signature = (
-        func.structured_input_signature)
-    return rebuilt_func
-
   def _execute_calibration(self, calibration_input_fn):
```

Comment:
Lint check requests for a docstring and this code isn't new in this PR, so I don't understand the Link error. But can you just add a docstring?